gengokit/template/NAME-service/NAME-server/server_main.gotemplate

```diff
@@ -60,19 +60,21 @@ func main() {
 	var service pb.{{GoName .Service.Name}}Server
 	{
 		service = handler.NewService()
-		// add service logging and metrics here
+		// Inject Service middlewares. See ../middlewares/service.go
+		service = middlewares.InjectServiceMiddlewares(service)
 	}
 
 	// Endpoint domain.
 	var (
 	{{range $i := .Service.Methods -}}
-		{{ToLower $i.Name}}Endpoint = svc.Make{{$i.Name}}Endpoint(service)
+			{{ToLower $i.Name}}Endpoint = svc.Make{{$i.Name}}Endpoint(service)
```

Comment:
Looks like a rogue tab got in here. Figuring that gofmt won't work on these files since they're not actually valid Go, so we need to watch formatting closely ourselves in the templates.